macOS/geotoad.rb

```diff
@@ -0,0 +1,21 @@
+class Geotoad < Formula
+  desc "Query tool to query the geocaching.com website"
+  homepage "https://github.com/steve8x8/geotoad"
+  url "https://github.com/steve8x8/geotoad/archive/3.25.4.tar.gz"
```

Comment:
I’ve created the repository/tap and sent you a transfer request. If you accept it, it will be under your account, and you edit that file instead.

If you accept, will make the necessary changes to the README for the new installation process, as well as delete this file (and enclosing directory) from this PR.